### ISSUE TITLE
(pattern) Disabled states refresh

### ISSUE THREAD

**Issue by connor-leech** [carbon-design-system/carbon-website#812]
### Overview

As the document is written, it stipulates that there are two ways a page element can be styled:

- Disabled
- Read-only

These variations are specific to forms and carry significant functionality and accessibility implications.

In forms, a **disabled state** is styled by the browser and is virtually invisible to the system. It is not read by a screen-reader and is not sent with the rest of the form when the form is submitted.

**Read-only states**, on the other hand, are visible to screen readers and the data is submitted along with the rest of the form.

The guidance currently uses these terms as stylistic descriptions even though they hold semantic HTML definitions.

We should remove the section on ready only and only cover disabled states as it pertains to the styling of a component.

We also need to be sure that disabled vs. read-only is covered on the form pattern page. Accessibility considerations should be included for both.

### Next steps

- [x] Update this pattern to follow the latest pattern template
- [x] Meet with Jan and Jillian to collect and review feedback
- [x] Work in Jan's copy edits
- [x] Add captions to all images
- [x] Check capitalization in images
- [x] Confirm alt text reflects image
- [x] Clearly state that disabled state is a visual style
- [x] Include specific WCAG accessibility reference regarding HTML tags for read-only and disabled in forms
- [x] Sharpen up "Hidden" section and include stipulation about administrator decisions.
- [x] Examples in text should reflect those in the images

### Future steps (v2)

- [ ] Remove section on read-only
- [ ] Convert comparison images to do/don't examples
- [ ] "There used to be a rule of thumb that a system should never show a disabled control or element unless it was obvious how the user would enable it." - Chris Connors
- [ ] If an option is disabled due to permissions, should it be hidden rather than disabled?
- [ ] Meet with Jan and Jeannie to review example images. Some are unnecessarily complex
- [ ] Contact SMEs to collect feedback and in-product examples
- [ ] Explore toggles/switches as they relate to disabled states. How should this behavior enable/disable components?
- [ ] Meet w/ Josh for accessibility content
- [ ] Check capitalization in images
- [ ] Confirm accuracy of states and styles in images (Anna or Lauren)
- [ ] Research composing a page with multiple types of progress indicators.
- [ ] Include development advice for loading order/orchestration.
- [ ] Contact SMEs for product examples and feedback.
